modules/olm-installing-opm.adoc

```diff
@@ -2,20 +2,27 @@
 //
 // * operators/olm-managing-custom-catalogs.adoc
 
+ifdef::openshift-origin[]
+:registry-image: quay.io/openshift/origin-operator-registry:4.6.0
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.6
+endif::[]
+
 [id="olm-installing-opm_{context}"]
 = Installing `opm`
 
-You can install the `opm` CLI tool on your workstation.
+You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
```

Comment:
@adellape sorry to forget to tell you one bug about opm on win10 when last review. 
there is issue https://bugzilla.redhat.com/show_bug.cgi?id=1883773 for opm on win10.

Reply:
OK, thank you. I can add it as a "Known issue" to the OCP 4.6 release notes in a separate PR.

Reply:
@adellape OK, got it, thanks

Reply:
Added here: https://github.com/openshift/openshift-docs/pull/25624/files#diff-3210eb8b77dfc42cf53d0a68f1a23845adfe8520940415e3c3d4abfc22f5ed68R971

Reply:
@adellape Thanks